### PR TITLE
Pull req 6

### PR DESCRIPTION
**Pull request by AlexD1337**
### Description

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,4 +5,5 @@
 <part-4></part-4>
 
 <!-- Advanced version control flow with Pull Request (PR), in group -->
-<part-5-pr></part-5-pr>
\ No newline at end of file
+<part-5-pr></part-5-pr>
+<part-6-pr></part-6-pr>
```
Comment (sibala) [inline index.html:9]: Great addition. Have you though considered that we might solve this issue as we did in the previous pages?

**Review by sibala — CHANGES_REQUESTED** [review]
Everything looks good. Had just on question about the code?

File: index.html (hunk 1 above)
Comment (AlexD1337) [reply index.html:9]: Yes I did but I find my solution more effective this time! Thanks for the review :)

Comment (sibala) [reply index.html:9]: Ok great. Thank you for clarifying.

**Review by sibala — APPROVED** [review]
Looks great. Well done :)

**Pull request merged**